pandas/tests/frame/test_operators.py

```diff
@@ -343,9 +343,15 @@ def test_logical_with_nas(self):
         expected = Series([True, True])
         assert_series_equal(result, expected)
 
+    def test_pos(self):
+        try:
+            assert_frame_equal(+self.frame, self.frame.apply(lambda x: +self.frame))
```

Comment:
dont' test like this. instead use a couple of sample frames that have singular dtypes (can be a single column). test the valid ones via ``assert_frame_equal(-frame, -1 * frame)``. The invalid ones should raise.